### PR TITLE
fix: add the Hebrew plural formula

### PR DESCRIPTION
**Pull request by yarons**
Based on the CLDR rule, in the gettext form:
nplurals=4; plural=(n == 1) ? 0 : ((n == 2) ? 1 : ((n > 10 && n % 10 == 0) ? 2 : 3))

@asbiin Most of the mentioned sources show the former rule, not the current one.

### CODE REVIEW

File: resources/js/pluralization.js [hunk 1]
```diff
@@ -30,11 +30,17 @@ function pluralE (choice, choicesLength) {
   number = (number == 0) ? 0 : ((number == 1) ? 1 : ((number == 2) ? 2 : (((number % 100 >= 3) && (number % 100 <= 10)) ? 3 : (((number % 100 >= 11) && (number % 100 <= 99)) ? 4 : 5))));
   return Math.min(number, choicesLength - 1);
 }
+function pluralF (choice, choicesLength) {
+  let number = Math.abs(choice);
+  number = (number == 1) ? 0 : (((number == 2) ? 1 : ((number < 10 && number % 10 == 0)) ? 2 : 3);
```
Comment (asbiin) [inline resources/js/pluralization.js]: You have too much parenthesis:
```suggestion
  number = (number == 1) ? 0 : ((number == 2) ? 1 : (number < 10 && number % 10 == 0) ? 2 : 3);
```

**Comment by yarons** [comment]
Yup, thanks, sorry for the trouble, feel free to merge 😅

File: resources/js/pluralization.js (hunk 1 above)
Comment (yarons) [reply resources/js/pluralization.js]: Sure, thanks, ready to merge?

Comment (asbiin) [reply resources/js/pluralization.js]: This is still unresolved.
You'll have to rebuild the assets too, but I can do it, with this fix too. However, you'll have to sign the CLA please.

**Comment by yarons** [comment]
Done.

**Comment by asbiin** [comment]
Thank you @yarons!
feal free to change translations now, but this change only affect vue.js client side translations, with curly brackets {}

**Pull request merged**